src/SmartComponents/modules/actions.js

```diff
@@ -47,9 +47,35 @@ function toggleFilterDropDown() {
 }
 
 function toggleFactSort(sortType) {
+    let newSort;
+
+    if (sortType === 'asc') {
```

Comment:
Not required, but we might want to consider moving these to constants that can be referenced.

Reply:
I think it would help me with readability if `sortType` was something like `previousSort` or `currentSort` to indicate the value that is coming in is the old value that we need to translate to the new.